src/main/java/teammates/common/datatransfer/FeedbackSessionAttributes.java

```diff
@@ -301,6 +301,30 @@ public boolean isClosingWithinTimeLimit(int hours) {
                && differenceBetweenDeadlineAndNow >= hours - 1
                && differenceBetweenDeadlineAndNow < hours;
     }
+    
+    public boolean isClosedWithinPastHour() {
+        Calendar now = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
+        // Fix the time zone accordingly
+        now.add(Calendar.MILLISECOND,
+                (int) (60 * 60 * 1000 * timeZone));
+
+        Calendar start = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
+        start.setTime(startTime);
+
+        Calendar deadline = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
+        deadline.setTime(endTime);
+
+        long nowMillis = now.getTimeInMillis();
+        long deadlineMillis = deadline.getTimeInMillis();
+        long differenceBetweenNowAndDeadline = (deadlineMillis - nowMillis) / (60 * 60 * 1000);
+
+        // If now and start are almost similar, it means the feedback session
+        // is open for only 24 hours.
+        // Hence we do not send a reminder e-mail for feedback session.
```

Comment:
This makes sense for closing reminder, but is it so for closed reminder? A closed feedback session is closed regardless of the amount of time it is opened.